src/main/java/codechicken/nei/ItemQuantityField.java

```diff
@@ -44,6 +44,7 @@ public void draw(int mousex, int mousey) {
 
         if (!focused() && intValue() == 0) {
             field.setText(translate("itempanel.quantity.default"));
+            field.setCursorPositionZero();
```

Comment:
can you explain what is this supposed to do ? does that fix a bug ?

Reply:
this fixes the display the placeholder from the beginning
![image](https://github.com/user-attachments/assets/e60cf9ed-0312-4976-b9e8-00084de9d1e6)